src/components/VersionChooser/index.spec.tsx

```diff
@@ -317,6 +325,37 @@ describe(__filename, () => {
     );
   });
 
+  it('dispatches setCurrentBaseVersionId() when the old version changes', () => {
+    const addonId = nextUniqueId();
+    const baseVersionId = nextUniqueId();
+    const headVersionId = nextUniqueId();
+
+    const store = configureStore();
+    _loadVersionsList(store, addonId, fakeVersionsList);
+    const dispatchSpy = spyOn(store, 'dispatch');
+
+    const selectedVersion = String(nextUniqueId());
+
+    const root = render({
+      addonId,
+      baseVersionId,
+      headVersionId,
+      store,
+    });
+
+    const onChange = root
+      // Retrieve the `VersionSelect` component with this `className`.
+      .find({ className: styles.baseVersionSelect })
```

Comment:
For some crazy reason, this find syntax will preserve the type of the `onChange` prop so that TS knows it's ok to call it as a function. There's another test in this suite that uses it for the same reason.